include/libtorrent/session.hpp

```diff
@@ -308,7 +308,7 @@ namespace aux {
 			char if_string[100];
 
 			if (listen_interface == nullptr) listen_interface = "0.0.0.0";
-			std::snprintf(if_string, sizeof(if_string), "%s:%d", listen_interface, listen_port_range.first);
```

Comment:
why remove `std::`?

Reply:
baaically this doesn't compile on windows (msvc 14.0)

Reply:
I believe that it's not about msvc 14.0, but because of these lines:
```
#ifndef TORRENT_NO_DEPRECATE
#include "libtorrent/fingerprint.hpp"
#include <cstdio> // for snprintf
#endif
```
Instead of removing `std::`, try moving `#include <cstdio>` out of `#ifndef` block.

Reply:
I tried moving it out of the ifndef with no luck :(

Reply:
are you sure you're building in C++11 mode? ``std::snprintf()`` was introduced with C++11.

Reply:
wow you are right, chaning for using msvc : 14.0 : : /std:c++11 fix the issue, I will update the docs accordly :)